### ISSUE TITLE
server.shtml Unmatched end tag

### ISSUE THREAD

**Issue by MaximMartel** [wet-boew/cdts-sgdc#77]
Moved from GCCode: https://gccode.ssc-spc.gc.ca/iitb-dgiit/nw-ws/sgdc-cdts/issues/105

@StdGit created the issue

https://ssl-templates.services.gc.ca/app/cls/WET/gcweb/v4_0_32/cdts/samples/server.shtml

**Comment by MaximMartel** [comment]
@shawnthompson commented:

@StdGit  can you please give more detail for this issue?

**Comment by nschonni** [comment]
http://validator.w3.org/nu/?doc=https%3A%2F%2Fssl-templates.services.gc.ca%2Fapp%2Fcls%2FWET%2Fgcweb%2Fv4_0_32%2Fcdts%2Fsamples%2Fserver.shtml

Has the following errors
```
    Error: End tag div seen, but there were open elements.

    From line 58, column 4; to line 58, column 9

    </div>↩			</div>↩		</h

    Error: Unclosed element header.

    From line 45, column 6; to line 45, column 13

     -->↩					<header>↩			<d

    Error: Stray end tag header.

    From line 59, column 3; to line 59, column 11

    	</div>↩		</header>↩↩		</

    Error: Stray end tag div.

    From line 61, column 3; to line 61, column 8

    eader>↩↩		</div>↩		<!-
```

**Comment by patricehernert** [comment]
@MaximMartel Could you confirm this issue is still valid for version 4.0.39 ? Thanks.

**Comment by ahmad-shahid** [comment]
This has been fixed for v4.0.39. See this: https://validator.w3.org/nu/?doc=https%3A%2F%2Fssl-templates.services.gc.ca%2Fapp%2Fcls%2FWET%2Fgcweb%2Fv4_0_39%2Fcdts%2Fsamples%2Fserver.shtml
